README.md

```diff
@@ -1,24 +1,46 @@
-# What is `kedro-examples`?
-
-This repo contains links to all official and community created example Kedro projects as well as the complete version of the [spaceflights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/02_tutorial_template.html).
-
-## What will I find in this repository?
-
-- We recommend starting with [`kedro-tutorial`](https://github.com/quantumblacklabs/kedro-examples/tree/master/kedro-tutorial), which is the complete version of the [Space Flights tutorial](https://kedro.readthedocs.io/en/stable/03_tutorial/02_tutorial_template.html) described in the Kedro [documentation](https://kedro.readthedocs.io) and includes the data necessary to run the project. Some of the old code examples are available in the following links:
-
-  - [Kedro 0.14.0](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.14.0)
-  - [Kedro 0.15.0](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.15.0)
-  - [Kedro 0.15.5](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.15.5)
-  - [Kedro 0.15.8](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.15.8)
-  - [Kedro 0.16.1](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.1)
-  - [Kedro 0.16.4](https://github.com/quantumblacklabs/kedro-examples/tree/example/0.16.4)
-
-
-## Where can I find more learning material and `kedro-examples`?
-
-The [`Kedro-Training`](https://github.com/quantumblacklabs/kedro-training) repository contains documentation that you can use to deliver Kedro training.
-
-These repositories contain the official Kedro tutorial projects, also known as starters:
+# What is `kedro-community`?
+
+This repo contains links to articles, podcasts and videos about Kedro, as well as, community created examples of Kedro projects. If you're looking to engage with the community then we suggest checking out the [Kedro Community Discourse](https://discourse.kedro.community/) channel.
+
+## Where can I find articles, podcasts and videos about Kedro? 
```

Comment:
This section is quite long, so people might not notice there's a section "Where can I find more example projects using Kedro?" underneath. Maybe we can swap these two sections?

Reply:
I like this suggestion, will do!